### ISSUE TITLE
framework-agnostic?

### ISSUE THREAD

**Issue by wey-gu** [SwordElucidator/nebula-carina#2]
This is not a request, just if possible, decoupling the existing framework could help more python users :-P

But I would like to learn Pydantic for nebula-carina!!!

👍🏻

**Comment by SwordElucidator** [comment]
Yeah pydantic is quite easy to handle with the model/ serialization etc. It is almost like a light-weighted python3-styled package rather than a web framework. You can use it in flask / django, but it is most welcomed in fastapi. I will make the settings module more like the native settings usage in django later.
